test/src/main/java/org/apache/accumulo/test/fate/zookeeper/FateIT.java

```diff
@@ -243,27 +244,39 @@ public void testTransactionStatus() throws Exception {
       assertEquals(IN_PROGRESS, getTxStatus(zk, txid));
       // tell the op to exit the method
       finishCall.countDown();
-      // Check that it transitions to SUCCESSFUL
-      TStatus s = getTxStatus(zk, txid);
-      while (s != SUCCESSFUL) {
-        s = getTxStatus(zk, txid);
```

Comment:
This is the line it kept failing on with NoNode. NoNode was expected on line 256, but it could just as easily have happened here on line 249.